airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresQueryUtils.java

```diff
@@ -28,21 +28,23 @@
  */
 public class PostgresQueryUtils {
 
+  public record TableBlockSize(Long tableSize, Long blockSize) { }
+
   private static final Logger LOGGER = LoggerFactory.getLogger(PostgresQueryUtils.class);
 
   public static final String NULL_CURSOR_VALUE_WITH_SCHEMA_QUERY =
       """
         SELECT
           (EXISTS (SELECT FROM information_schema.columns WHERE table_schema = '%s' AND table_name = '%s' AND is_nullable = 'YES' AND column_name = '%s'))
         AND
-          (EXISTS (SELECT from \"%s\".\"%s\" where \"%s\" IS NULL LIMIT 1)) AS %s
+          (EXISTS (SELECT from "%s"."%s" where "%s" IS NULL LIMIT 1)) AS %s
```

Comment:
why these changes?

Reply:
To remove spot check warnings.
No need to escape inside """ block